adam/adam_6224_ds/adam_6224.py

```diff
@@ -0,0 +1,355 @@
+
+# -*- coding: utf-8 -*-
+#
+# This file is part of the dev-solaris-adam project
+#
+#
+#
+# Distributed under the terms of the LGPL license.
+# See LICENSE.txt for more info.
+
+""" Adam6224
+It is a definicion of a class used to control ADAM-6224 controller via Modbus/TCP.
+"""
+
+__all__ = ["Adam6224", "main"]
+
+# PyTango imports
+import tango
+from tango import DebugIt
+from tango.server import run
+from tango.server import Device, DeviceMeta
+from tango.server import attribute, command
+from tango.server import class_property, device_property
+from tango import AttrQuality, AttrWriteType, DispLevel, DevState
+# Additional import
+# PROTECTED REGION ID(Adam6224.additionnal_import) ENABLED START #
+from pymodbus.client.sync import ModbusTcpClient
+from pymodbus.exceptions import ModbusException
+#mport time
+# PROTECTED REGION END #    //  Adam6224.additionnal_import
+
+
+class Adam6224(Device):
+    """
+    It is a definicion of a class used to control ADAM-6224 controller via Modbus/TCP.
```

Comment:
Please provide a better description here. You can write about the types and quantities of its connections.